### DISCUSSION TITLE
Call service to turn on a switch without data

### DISCUSSION THREAD

**Discussion by n3de** [zachowj/node-red-contrib-home-assistant-websocket#1408] [Q&A] [answered]
Hey,

I set up a flow to turn on and off an switch. I run a few updates and I get the message, that the property data is invalid:

![image](https://github.com/zachowj/node-red-contrib-home-assistant-websocket/assets/81101039/23cb9e73-7b2e-4317-b8ac-f8d5d8a2cbac)

![image](https://github.com/zachowj/node-red-contrib-home-assistant-websocket/assets/81101039/a9e4462a-1f23-47bd-b7fe-95298543020d)

I checked the cook book, but I do not understand, what to fill into the data field? It works in the past without the error message.

**Comment by zachowj** [comment, marked as answer]
Are you using the latest version of this package? There was a bug with the data field several releases ago.

**Reply by n3de** [reply]
I get the error message after I updated node-red to 3.1.9. But after yours reply, I checked node-red and I doesn't know, that I have to update all plugins manuelly. I thougt it would be updated with the node-red update. I updated everythink and it is working now. Thanks for the hint!
